src/components/AppStyle.tsx

```diff
@@ -65,8 +65,10 @@ const theme = {
     },
   },
   font: {
-    family: "Lato",
-    resource: "https://fonts.googleapis.com/css2?family=Lato&display=swap",
+    // Lato does not provide Greek characters so Carlito (which is derived from Lato
+    // and provides Greek + Latin + Cyrillic) is used as a fallback.
+    family: "Lato, Carlito, sans-serif",
+    resource: "https://fonts.googleapis.com/css2?family=Lato&family=Carlito&display=swap",
```

Comment:
does this change affect the appearance of any text other than the Greek characters?

Reply:
Yes, kinda. It won't affect standard text characters, as they'll still default to Lato. The font family attribute here acts as a series of fallbacks, where if a character is not found in a font the browser will try to use the next one, eventually going to some predefined default (I think it's usually Times New Roman).

Carlito provides a LOT more glyphs than Lato does, including groups other than Greek characters.
![image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/83e1cd53-ab17-481b-a70c-c3bbdc930362)
![image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/f71498c1-aa27-49d4-ba17-caf24322e81e)

There *are* versions of Lato that do have complete character sets (https://www.latofonts.com/lato-free-fonts/#download) but we'd have to include that in the website files and it adds like 250 kB to the webpage download size. It also won't hook into the browser caching that Google Fonts provides. I specifically chose Carlito for this use case (mu) because it looks relatively consistent and it's easy to add in via Google Fonts, but if we start to see weird font deviances this might be the better option.

Reply:
Okay wait actually I'm just going to use the Lato extended font I don't know why I didn't think about this earlier